package.json

```diff
@@ -20,5 +20,12 @@
   "bugs": {
     "url": "https://github.com/CoderDojo/cp-translations/issues"
   },
-  "homepage": "https://github.com/CoderDojo/cp-translations#readme"
+  "homepage": "https://github.com/CoderDojo/cp-translations#readme",
+  "devDependencies": {
+    "mocha": "^4.0.1"
+  },
+  "dependencies": {
+    "ejs": "^2.5.7",
+    "email-templates": "^2.0.1"
```

Comment:
why dep and not dev dep?

Reply:
Because I'm an idiot 😛

Reply:
also did you not say you didnt need ejs or do i just need more coffee?